src/client/theme/SearchPage/SearchPage.tsx

```diff
@@ -1,33 +1,31 @@
-declare let _paq: Array<[string, string, boolean, number]>;
-declare let gtag: any;
-
-import React, { useCallback, useEffect, useMemo, useState } from "react";
-import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
-import { usePluginData } from "@docusaurus/useGlobalData";
-import Layout from "@theme/Layout";
-import Head from "@docusaurus/Head";
-import Link from "@docusaurus/Link";
-
-import { GlobalPluginData } from "docusaurus-plugin-search-local";
-import useSearchQuery from "../hooks/useSearchQuery";
-import { fetchIndexes } from "../../utils/fetchIndexes";
-import { SearchSourceFactory } from "../../utils/SearchSourceFactory";
-import { SearchDocument, SearchResult, SearchSourceFn } from "../../../types";
-import { highlight } from "../../utils/highlight";
-import { highlightStemmed } from "../../utils/highlightStemmed";
-import { getStemmedPositions } from "../../utils/getStemmedPositions";
-import LoadingRing from "../LoadingRing/LoadingRing";
-import { simpleTemplate } from "../../utils/simpleTemplate";
-import ErrorBoundary from "../ErrorBoundary";
-
-import styles from "./SearchPage.module.css";
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
```

Comment:
It looks like my local prettier config is wanting single quotes, but this repo is using the default. Is that expected?